src/app/components/pool/action/add/pool-create-basic-dialog.component.ts

```diff
@@ -38,6 +39,14 @@ export class PoolCreateBasicDialogComponent extends DynamicForm<Pool, PoolCreate
         return this._osControl.value && this._osControl.value.cloudServiceConfiguration;
     }
 
+    public get isSelectedContainerImageDeprecated() {
+        if (this._osControl.value && this._osControl.value.virtualMachineConfiguration) {
+            const selectedImage = this._osControl.value.virtualMachineConfiguration.imageReference.sku;
+            return DeprecatedContainerImages.includes(selectedImage);
```

Comment:
I'm not sure it matters (can't remember if pool properties are case-preserving), but we should probably do a case-insensitive comparison.

Reply:
Thanks for catching that!